src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -53,7 +53,7 @@
 </div>
 <ng-content select="clr-dg-footer"></ng-content>
 <div class="datagrid-spinner" *ngIf="loading">
-    <div class="spinner spinner-md">Loading...</div>
+    <clr-spinner class="spinner-md">Loading...</clr-spinner>
```

Comment:
Similar to the progress bar suggestion, let's take the opportunity to encapsulate the classes in the template and provide attribute style hooks.

`<clr-spinner medium inverse inline></clr-spinner>`